indico/modules/events/contributions/blueprint.py

```diff
@@ -27,6 +27,8 @@
                       virtual_template_folder='events/contributions', url_prefix='/event/<confId>')
 
 _bp.add_url_rule('/manage/contributions/', 'manage_contributions', RHContributions)
+_bp.add_url_rule('/manage/contributions/customize', 'customize_contrlist',
```

Comment:
well yeah, should be renamed :P I was following the reglist principal, but you're right, it kinda sucks